protokube/pkg/protokube/kube_boot.go

```diff
@@ -154,8 +155,15 @@ func enableKubelet() error {
 	// TODO: Check/log status of kubelet
 	// (in particular, we want to avoid kubernetes/kubernetes#40123 )
 	glog.V(2).Infof("ensuring that kubelet systemd service is running")
-	cmd := exec.Command("systemctl", "start", "--no-block", "kubelet")
+	cmd := exec.Command("systemctl", "status", "--no-block", "kubelet")
 	output, err := cmd.CombinedOutput()
```

Comment:
Would be great to glog the output here, but I'll send a follow-on PR rather than nit-picking :-)